docs/howto_firewall.md

```diff
@@ -3,11 +3,13 @@ MicroShift does not require a firewall for normal operation. However, it is reco
 
 It is mandatory to allow MicroShift pods the access to the internal CoreDNS and API servers.
 > Users may choose to change the pod IP range to be different from the default `10.42.0.0/16` setting. This must be reflected in the firewall configuration.
+> Users may choose to change the service IP range to be different from the default `10.43.0.0/16` setting.
```

Comment:
since we don't have firewall rules associated with service IP range, maybe just add it in the description of 169.254.169.1, indicating that this special IP needs to be enabled for accessing service backed by host endpoints?

Reply:
Changed the description and ditched the note.